templates/rhpam70-authoring.yaml

```diff
@@ -188,6 +188,15 @@ parameters:
   description: Fully qualified URL to a Maven repository. If unspecified, will fall back to Business Central service.
   name: MAVEN_REPO_URL
   required: false
+- displayName: Maven repository username
+  description: Username to access the Maven repository.
+  name: MAVEN_REPO_USERNAME
+  required: true
```

Comment:
Same question I did on the rhdm tempaltes, should it be required as we have default values if empty?

Reply:
Circling back, I agree with Filippe that maven repo username should not be required.